### PR TITLE
add 10 users to cluster; add optional...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

add 10 users to cluster; add optional kubeadmin oc login; add counts of how long provisioning actually takes

Change-Id: Ia7917c9eb39d0cbc82ecfc9891a3d10c5238432d
Signed-off-by: Nick Boldt <nickboldt@gmail.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: product/installDevSpacesFromLatestIIB.sh [hunk 1]
```diff
@@ -32,14 +32,18 @@ usage() {
...
 3. Install the Dev Spaces Operator from the new CatalogSource
-4. Create a CheCluster to install Dev Spaces
+4. Create 10 dummy OpenShift users from user1 to user10
+5. Create a CheCluster to install Dev Spaces
 
 Usage: $0 [OPTIONS]
 
 Options:
+  --kubeadmin         : If not already connected to an OCP instance, use this kubeadmin password
+  --openshift         : If not already connected to an OCP instance, use this api.cluster:6443 url
```
Comment (amisevsk) [inline product/installDevSpacesFromLatestIIB.sh]: Maybe `--ocp-url`? The combo of `kubeadmin` and `openshift` is throwing me and I initially read them as `--kubernetes` and `--openshift``

Maybe also a token parameter (passed via `oc login -t <token>`) would be useful (or maybe that'd be overengineering a utility script :shrug:)

File: product/installDevSpacesFromLatestIIB.sh [hunk 2]
```diff
@@ -52,26 +56,38 @@ EOF
...
+  if [[ $KUBE_PWD ]] && [[ $OCP_URL ]]; then
+    oc login ${OCP_URL} --username=kubeadmin --password=${KUBE_PWD}
+  fi
   if ! oc whoami > /dev/null 2>&1; then
     errorf "Not logged into an OpenShift cluster"
     exit 1
   fi
-  BREW_TOKENS_NUM="$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s | jq -r 'length')"
-  if [[ "$BREW_TOKENS_NUM" == "0" ]]; then
+  TOKENS_YAML=$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s)
```
Comment (amisevsk) [inline product/installDevSpacesFromLatestIIB.sh:70]:
```suggestion
  TOKENS_JSON=$(curl --negotiate -u : https://employee-token-manager.registry.redhat.com/v1/tokens -s)
```

**Review by amisevsk — APPROVED** [review]

File: product/installDevSpacesFromLatestIIB.sh (hunk 1 above)
Comment (amisevsk) [reply product/installDevSpacesFromLatestIIB.sh]: Already fixed I see :)

**Comment by nickboldt** [comment]
```
$➔ ./installDevSpacesFromLatestIIB.sh -t 3.0 -kp ******** -os api.*******:6443
The server uses a certificate signed by an unknown authority.
You can bypass the certificate check, but any data you send to the server could be intercepted by others.
Use insecure connections? (y/n): y

Login successful.

You have access to 65 projects, the list has been suppressed. You can list all projects with 'oc projects'

Using project "default".
Detected OpenShift version v4.10
Found latest IIB registry-proxy.engineering.redhat.com/rh-osbs/iib:234381
Using iib image brew.registry.redhat.io/rh-osbs/iib:234381 mirrored from registry-proxy.engineering.redhat.com/rh-osbs/iib:234381
Login Succeeded!
secret/pull-secret data updated
imagecontentsourcepolicy.operator.openshift.io/brew-registry created
catalogsource.operators.coreos.com/iib-testing-catalog created
subscription.operators.coreos.com/devspaces created
Waiting for CheCluster CRs to be available in the cluster. Timeout is 3 minutes.
............................ 81 s elapsed
checluster.org.eclipse.che/devspaces created
secret/htpass-secret created
oauth.config.openshift.io/cluster patched
Waiting for Dev Spaces to install in the cluster. Timeout is 15 minutes.
................................................. 240 s elapsed
```

**Pull request merged**